src/ui/controls/dropdown/style.less

```diff
@@ -47,7 +47,10 @@
         overflow: hidden;
         position: absolute;
         width: 100%;
-
+         max-height: 10rem;                   // added
+         overflow-y: auto;                    // added
+         -webkit-overflow-scrolling: touch;  // added
+         z-index: 1000;
```

Comment:
Why `z-index` is necessary? The number should be the lowest possible.

Properties should be separated from the nested selector with a new line.